.github/workflows/sycl_precommit.yml

```diff
@@ -27,53 +27,8 @@ permissions:
   contents: read
 
 jobs:
```

Comment:
Do I understand it correctly that we leave post-commit as is because the method to detect impacted tests is different?

Reply:
We don't do such detection/filtering there at all.

Reply:
I know and my question is "why?". It looks like a low hanging fruit to speed-up CI.

Reply:
Two reasons:
1) I don't feel comfortable doing the switch yet.
2) I don't think it's necessarily the right choice. Imagine an xptifw test failing. We change that part so rarely that we won't be running it for weeks if not months and it would be easy for the bugs to slip through.

Reply:
Commit1 - green
Commit2 - red, regression
CommitN..M - green, because didn't touch that component, regression is still present

Reply:
Commit 2 - red, regression <-- this is where the bug is caught, so I don't how it can "slip through".

Are you trying to say that the gatekeeper who merged commit 2 can skip reporting this issue and failing post-commits for commit N..M expose this issue to other gatekeepers, who will report it?